src/pybind/mgr/ansible/module.py

```diff
@@ -642,49 +687,106 @@ def _launch_operation(self, ansible_operation):
         self.all_completions.append(ansible_operation)
 
     def verify_config(self):
-        """ Verify configuration options for the Ansible orchestrator module
-        """
-        client_msg = ""
+        """Verify mandatory settings for the module and provide help to
+           configure properly the orchestrator
+        """
+
+        the_crt = None
+        the_key = None
+
+        # Retrieve TLS content to use and check them
+        # First try to get certiticate and key content for this manager instance
+        # ex: mgr/ansible/mgr0/[crt/key]
+        self.log.info("Tying to use configured specific certificate and key"
+                      "files for this server")
+        the_crt = self.get_store("{}/{}".format(self.get_mgr_id(), "crt"))
+        the_key = self.get_store("{}/{}".format(self.get_mgr_id(), "key"))
+        if the_crt is None or the_key is None:
+            # If not possible... try to get generic certificates and key content
+            # ex: mgr/ansible/[crt/key]
+            self.log.warning("Specific tls files for this manager not "\
+                             "configured, trying to use generic files")
+            the_crt = self.get_store("crt")
+            the_key = self.get_store("key")
+
+        if the_crt is None or the_key is None:
+            self.status_message = "Not running: no client certificate "\
+                                  "configured. Is needed to load Ansible "\
+                                  "Runner service certificate and key using "\
+                                  "'ceph ansible "\
+                                  "set-ssl-certificate-key/set-ssl-certificate"
```

Comment:
Maybe we can improve the error message. What about:

> No client certificate configured. Please set Ansible Runner Service client certificate and key:
> 
> `ceph ansible set-ssl-certificate-{key,certificate} -i <file>`

?

Reply:
Thx! Done